packages/gui/components/Select/Dropdown.tsx

```diff
@@ -18,6 +20,17 @@ const Dropdown: FC<Props> = ({ config }) => {
 	const { selected, items, title, getRequiredFields, onSelect } =
 		config.context as SelectionContext<object>;
 
+	const [searchText, setSearchText] = useState('');
```

Comment:
not efficient yet, use `debounce` (for function instead of state) from lodash and then `useMemo` to memorize filtered item.